packages/grid/components.ts

```diff
@@ -0,0 +1,14 @@
+import { StarryUIComponentDefinition } from '@starryui/traits'
+import { grid } from '.'
+import { withTextContent } from '@starryui/traits'
+import { applyTheme } from '@starryui/theme'
+
+export const buttonDefinition: StarryUIComponentDefinition = {
+ title: 'grid',
+ exampleSource: `const themedGrid = applyTheme(theme, grid)
+return themedGrid.add(withTextContent('A grid'))()`,
```

Comment:
`@starryui/grid` does not support `textContent` trait - this should be a working example of the grid that will appear in the component documentation. Instead, use the new `withCellDimensions` and `withCellRender` here to create and test your grid component in the documentation. You should be running `npm run docs:watch` to test this.